src/Middleware/SentryIntegration.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Sentry\Middleware;
+
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\MiddlewareInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Sentry\State\HubInterface;
+use Throwable;
+
+final class SentryIntegration implements MiddlewareInterface
```

Comment:
MiddlewareInterface requires `psr/http-server-middleware` in composer's `require`.

Reply:
Done